x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/documents.tsx

```diff
@@ -5,8 +5,107 @@
  * 2.0.
  */
 
-import React from 'react';
+import React, { useEffect, ChangeEvent } from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import { SearchHit } from '@elastic/elasticsearch/lib/api/types';
+
+import {
+  EuiFieldSearch,
+  EuiTitle,
+  EuiSpacer,
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { Result } from '../../../shared/result/result';
+
+import { DocumentsLogic } from './documents_logic';
+import { IndexNameLogic } from './index_name_logic';
 
 export const SearchIndexDocuments: React.FC = () => {
-  return <>Documents</>;
+  const { indexName } = useValues(IndexNameLogic);
+  const { simplifiedMapping, results } = useValues(DocumentsLogic);
+  const { makeRequest, makeMappingRequest, setSearchQuery } = useActions(DocumentsLogic);
+
+  useEffect(() => {
+    makeRequest({ indexName, query: '' });
+    makeMappingRequest({ indexName });
+  }, [indexName]);
+
+  const resultToField = (result: SearchHit) => {
+    if (simplifiedMapping && result._source && !Array.isArray(result._source)) {
+      if (typeof result._source === 'object') {
+        return Object.entries(result._source).map(([key, value]) => {
+          return {
+            fieldName: key,
+            fieldType: simplifiedMapping[key]?.type ?? 'object',
+            fieldValue: JSON.stringify(value, null, 2),
+          };
+        });
+      }
+    }
+    return [];
+  };
+
+  return (
+    <EuiPanel hasBorder={false} hasShadow={false}>
+      <EuiSpacer />
+      <EuiFlexGroup direction="column">
+        <EuiFlexItem>
+          <EuiFlexGroup direction="row" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <EuiTitle>
+                <h2>
+                  {i18n.translate('xpack.enterpriseSearch.content.searchIndex.documents.title', {
+                    defaultMessage: 'Browse documents',
+                  })}
+                </h2>
+              </EuiTitle>
+            </EuiFlexItem>
+            <EuiFlexItem>
```

Comment:
This item needs a maxWidth so there's a good amount of space between Browse documents and this field

Reply:
I will make them 50/50 with flex, so there is space in between title and search field

Reply:
That doesn't match the designs:
<img width="881" alt="Screenshot 2022-07-25 at 18 10 30" src="https://user-images.githubusercontent.com/94373878/180824915-cde4ccaa-1054-4263-8bce-785a8bfd9715.png">

Reply:
max-width also not the best solution as well then. Adding some spacing in between them instead?

Reply:
![Screenshot 2022-07-25 at 18 27 06](https://user-images.githubusercontent.com/1410658/180828048-71c577dc-10de-4d10-8c4b-3297c3a2e950.png)
![Screenshot 2022-07-25 at 18 27 11](https://user-images.githubusercontent.com/1410658/180828052-951151d6-d6b4-4632-a2e8-5d032756a5d8.png)